$NetBSD$

In cmake build, adjust paths to pkgsrc norms.  (Originally from
OpenBSD.)

The main point (but not using examples for sysconf) is filed upstream as:
https://github.com/eclipse/mosquitto/issues/1049

--- CMakeLists.txt.orig	2018-09-25 15:27:43.000000000 +0000
+++ CMakeLists.txt
@@ -15,6 +15,8 @@ set (VERSION 1.5.3)
 
 add_definitions (-DCMAKE -DVERSION=\"${VERSION}\")
 
+include(GNUInstallDirs)
+
 if (WIN32)
 	set (BINDIR .)
 	set (SBINDIR .)
@@ -30,17 +32,12 @@ if (WIN32)
 else (WIN32)
 	set (BINDIR bin)
 	set (SBINDIR sbin)
-	if ("${CMAKE_INSTALL_PREFIX}" STREQUAL /usr)
-		set (SYSCONFDIR /etc/mosquitto)
-	else ("${CMAKE_INSTALL_PREFIX}" STREQUAL /usr)
-		set (SYSCONFDIR etc/mosquitto)
-	endif ("${CMAKE_INSTALL_PREFIX}" STREQUAL /usr)
+	set (SYSCONFDIR "${DATAROOTDIR}/examples/mosquitto")
 	
 	set (LIBDIR lib${LIB_SUFFIX})
-	set (CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${LIBDIR}")
 	set (INCLUDEDIR include)
 	set (DATAROOTDIR share)
-	set (MANDIR "${DATAROOTDIR}/man")
+	set (MANDIR ${CMAKE_INSTALL_MANDIR})
 	set (SHAREDIR "${DATAROOTDIR}/mosquitto")
 	set (PKGCONFIGDIR "${LIBDIR}/pkgconfig")
 endif (WIN32)
